contrib/scripts/kind.sh

```diff
@@ -163,6 +163,12 @@ if [ "${xdp}" = true ]; then
   done
 fi
 
+# Replace "forward . /etc/resolv.conf" in the coredns cm with "forward . 8.8.8.8".
+# This is required because in case of BPF Host Routing we bypass iptables thus
+# breaking DNS. See https://github.com/cilium/cilium/issues/23330
+NewCoreFile=$(kubectl get cm -n kube-system coredns -o jsonpath='{.data.Corefile}' | sed 's,forward . /etc/resolv.conf,forward . 8.8.8.8,' | sed -z 's/\n/\\n/g')
+kubectl patch configmap/coredns -n kube-system --type merge -p '{"data":{"Corefile": "'"$NewCoreFile"'"}}'
```

Comment:
Is this command synchronous? If not, is there a way to wait for the change to be done?

Reply:
What do you mean exactly? I think that after this command is finished, the `coredns` CM will not expose the old value. The coredns or any other non-hostns pods are not running at this stage yet